Makefile

```diff
@@ -74,6 +74,16 @@ else
   PLUGIN_OPTS := 
 endif
 
+# Reproducible builds mode
+GO_REPRO_VERSION := $(shell cat .go_repro_version)
+GO_REPRO_MAJOR := $(word 1,$(subst ., ,$(GO_REPRO_VERSION)))
+GO_REPRO_MINOR := $(word 2,$(subst ., ,$(GO_REPRO_VERSION)))
```

Comment:
`GO_REPRO_MINOR` seems to be unused.

Reply:
So it is... I can remove that on commit.